addons/nuxeo-ai-aws-core/src/main/java/org/nuxeo/ai/rekognition/RekognitionServiceImpl.java

```diff
@@ -67,6 +68,14 @@
 
     protected volatile AmazonRekognition client;
 
+    protected AIComponent aiComponent;
+
+    @Override
+    public void activate(ComponentContext context) {
+        super.activate(context);
+        aiComponent = Framework.getService(AIComponent.class);
```

Comment:
the order of the component load should be after AIComponent then

Reply:
I set them now in start method of the components. Is it ok to do that without handling loading order?

Reply:
Oh yes thats good: "Start the component. This method is called after all the components were resolved and activated"